mordecai3/elastic_utilities.py

```diff
@@ -273,16 +273,31 @@ def add_es_data(ex, conn, max_results=50, fuzzy=0, limit_types=False,
                              "fields": ['name', 'asciiname', 'alternativenames'],
                              "type": "phrase"}}
 
+    include_country_filter = None
+    if include_countries is not None and include_countries:
```

Comment:
Why both conditions?

Reply:
Checking for `None` & `non empty list`

Reply:
You can keep just the latter condition to handle both for this and `exclude_countries`.

Reply:
Sure, done